app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import Alert from '@/components/AlertNotification/AlertNotification';
+import { AlertVariants } from '@/components/AlertNotification/Alerts.enum';
+import { toast } from 'react-hot-toast';
+import { onWeeklyPickChange } from './WeekHelper';
+import { createWeeklyPicks } from '@/api/apiFunctions';
+import { parseUserPick } from '@/utils/utils';
+import { IWeeklyPicks } from '@/api/apiFunctions.interface';
+
+jest.mock('@/api/apiFunctions', () => ({
+  createWeeklyPicks: jest.fn(),
+}));
+
+jest.mock('react-hot-toast', () => ({
+  toast: {
+    custom: jest.fn(),
+  },
+}));
+
+describe('Week', () => {
+  const data = {
+    target: { value: 'Browns' },
+    preventDefault: jest.fn(),
+    stopPropagation: jest.fn(),
+  } as any;
```

Comment:
Shashi may have commented on this but I do not feel this should be a type of `any` since we're strictly typing everything else in our project.